src/Core/Commander/Interfaces/ApiInterface/ApiGlobe.js

```diff
@@ -246,7 +246,7 @@ define('Core/Commander/Interfaces/ApiInterface/ApiGlobe', [
     ApiGlobe.prototype.update = function() {
 
         //!\\ TEMP
-        this.scene.wait(0);
+        this.scene.step(0);
```

Comment:
Are the `//!\\ TEMP` comment still needed?
And I think it should be a `this.scene.notifyChange()`